packages/api/db/tables.sql

```diff
@@ -135,6 +154,22 @@ CREATE TABLE IF NOT EXISTS upload
     UNIQUE (user_id, source_cid)
 );
 
+CREATE VIEW admin_search as
+select
+  u.id::text as user_id,
+  u.github_id as username,
```

Comment:
```suggestion
  u.github as username,
```

Not sure if this is what you want, just to make sure. `user.github` is a numeric github ID, not the handle. We don't track the handle in web3.storage